corehq/sql_db/connections.py

```diff
@@ -192,12 +190,6 @@ def _populate_connection_map(self):
         if UCR_ENGINE_ID not in self.db_connection_map:
             self._add_django_db(UCR_ENGINE_ID, DEFAULT_DB_ALIAS)
 
-    def _populate_from_legacy_settings(self):
```

Comment:
does this mean that we're going to need to send out a commcare-cloud changelog?

Reply:
I don't think so - it's been a very long time since we made that change - I think before we even had `commcare-cloud`

Reply:
Ah. I was thinking of "legacy" as the refactor you just did a couple of days ago.